modules/local/grohmm/parametertuning/tests/main.nf.test.snap

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
What's with the empty snapshots? Same for transcriptcalling.

Reply:
They were empty before. They're really finicky(because of the underlying R packages) so if you look at the tests themselves they're actually failing.

Reply:
Sorry to be a pain, but wouldn't it be simpler to just not snapshot?